junit-jupiter-engine/src/test/java/org/junit/jupiter/api/AssertionsAssertArrayEqualsTests.java

```diff
@@ -1874,8 +1876,13 @@ void assertArrayEqualsDifferentNestedObjectArrays() {
 			expectAssertionFailedError();
 		}
 		catch (AssertionFailedError ex) {
+
 			assertMessageEquals(ex,
-				"array contents differ at index [3][0][1][0], expected: <5> but was: <" + differentElement + ">");
+				"array contents differ at index [3][0][1][0], expected: <5> but was: <"
+						// @sbrannen, @marcphillip: Do we want to display the
+						// default array str repr e.g. [I@3343c8b, or a
+						// list-like repr e.g. [1, 2, 3]?
+						+ Arrays.toString(differentElement) + ">");
```

Comment:
In cases like this, our goal is not to make Travis green for your PR but rather to have PRs and commits that focus on related tasks. If the static analysis makes us aware of a _bug_ in the code base, we should address that bug in a dedicated issue and not mix twenty different topics in a single PR.

The fact that the Travis build for a PR is momentarily broken is a nuisance, but a lesser evil.

So, yes, please move this to a separate PR.

TIA!